### PR TITLE
PTZ Presets Object token fix

### PR DESCRIPTION
**Pull request by Craytor**
Properly stores preset tokens as token, name as value

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "keypress": "^0.2.1",
     "mocha": "^10.0.0",
     "nimble": "^0.0.2",
-    "nyc": "^15.1.0"
+    "nyc": "^15.1.0",
+    "ip": "^2.0.1"
```
Comment (agsh) [inline package.json:65]: Hi! Why do you need an `ip` package in the dependencies?

Comment (Craytor) [reply package.json:65]: When I ran the development environment (`npm run mockserver`) for the tests, this was not found on my system so I included it in the dev dependencies. We can remove it if you didn't want to include it, but I had to manually run `npm install ip --save-dev`.

Comment (Craytor) [reply package.json:65]: Screenshot 2024-10-27 at 9 11 33 AM

Comment (agsh) [reply package.json:65]: Oh, that was 5 years ago :moyai:... and there is no need for that now :smile:
I'll merge your code right now and cleanup this part of the code later by myself

Comment (Craytor) [reply package.json:65]: Sounds good! Thanks for checking into this.

**Pull request merged**